scripts/docgen.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * This script is an adaptation of the build tooling from the official jsii-docgen codebase:
+ * https://github.com/aws/jsii-docgen/blob/d69453e6191d86761359d8ab28c4fd5e3e5ac2ad/src/index.ts#L11-L37
+ * The primary differences:
+ *  - only generates documentation if the module's `fqn` starts with `"@cdk-seed"`
```

Comment:
Is `fqn` here fully qualified name? If so, I didn't know what `fqn` was prior to looking it up and maybe someone new may not know that either and it may be best to write that out. If not, what is `fqn`? 🤔

Reply:
Also, this is not essential, just a nit

Reply:
I do believe that `fqn` means "Fully Qualified Name", but more importantly it is in reference to the `fqn` property of the `JsiiEntity` object.  I'll change it to this...

```suggestion
 *  - only generates documentation if the `JsiiEntity.fqn` starts with `"@cdk-seed"`
```